source/Calamari.AzureResourceGroup.Tests/AzureResourceGroupActionHandlerFixture.cs

```diff
@@ -69,37 +66,40 @@ public async Task Deploy_with_template_in_package()
         {
             var packagePath = TestEnvironment.GetTestPath("Packages", "AzureResourceGroup");
             await CommandTestBuilder.CreateAsync<DeployAzureResourceGroupCommand, Program>()
-                                    .WithArrange(context =>
-                                                 {
-                                                     AddDefaults(context);
-                                                     context.Variables.Add(SpecialVariables.Action.Azure.ResourceGroupDeploymentMode, "Complete");
-                                                     context.Variables.Add("Octopus.Action.Azure.TemplateSource", "Package");
-                                                     context.Variables.Add("Octopus.Action.Azure.ResourceGroupTemplate", "azure_website_template.json");
-                                                     context.Variables.Add("Octopus.Action.Azure.ResourceGroupTemplateParameters", "azure_website_params.json");
-                                                     context.WithFilesToCopy(packagePath);
-                                                 })
-                                    .Execute();
+                .WithArrange(context =>
+                {
+                    AddDefaults(context);
+                    context.Variables.Add(SpecialVariables.Action.Azure.ResourceGroupDeploymentMode, "Complete");
+                    context.Variables.Add("Octopus.Action.Azure.TemplateSource", "Package");
+                    context.Variables.Add("Octopus.Action.Azure.ResourceGroupTemplate", "azure_website_template.json");
+                    context.Variables.Add("Octopus.Action.Azure.ResourceGroupTemplateParameters", "azure_website_params.json");
+                    context.WithFilesToCopy(packagePath);
+                })
+                .Execute();
         }
 
         [Test]
         public async Task Deploy_with_template_inline()
         {
             var packagePath = TestEnvironment.GetTestPath("Packages", "AzureResourceGroup");
-            var paramsFileContent = File.ReadAllText(Path.Combine(packagePath, "azure_website_params.json"));
+            var templateFileContent = await File.ReadAllTextAsync(Path.Combine(packagePath, "azure_website_template.json"));
+            var paramsFileContent = await File.ReadAllTextAsync(Path.Combine(packagePath, "azure_website_params.json"));
```

Comment:
Using the async version and immediately await the result is the same as just calling the synchronised version. I just applied what Rider recommended here since it would be a more consistent default when two approaches are equivalent.